_posts/2018-06-08-pulp_qe_intern_update.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 2018 Pulp QE Intern Update
+author: Ragavendran Balakrishnan
+tags:
+  - pulpqe
+---
+
+
+How my journey started
+----------------
+
+The second week of February'18 was a very exciting period, as that's when I got to know I am gonna be a Red Hatter. Fast forward four months, Here I am joining as a QE in the pulp project in Red Hat. My role and responsibilities as a QE in pulp were very well delineated on a spreadsheet which I was presented on the first day. The first week, was very much a getting started phase. So here are the things which I learned. Pulp is a content management plugin that is a major cog in the Satellite Project machinery. It is responsible for syncing and managing contents in the yum repository (Oops! Its much more than that - It can work with contents in the python, docker, ostree, puppets and pretty much any files ). 
```

Comment:
We usually line-wrap at 100 chars instead of one really long line. Some people read code in editors that don't line wrap.